tokio-process/src/lib.rs

```diff
@@ -170,7 +171,41 @@ mod kill;
 /// Note that the `Child` type spawned is specific to this crate, and that the
 /// I/O handles created from this crate are all asynchronous as well (differing
 /// from their `std` counterparts).
```

Comment:
These docs could probably get updated now that we've got our own wrapper and we aren't extending `std::process::Command` :)

Reply:
I rewrote the docs, fixed doctests and copied over function docs from `std::process::Command`. I made the doctests a bit terse for `Command`, so that there aren't `#![feature(async_await)]`, `async` functions and the whole infrastructure everywhere (for example for trivial functions like `arg`, which are synchronous and just pass to the underlying `std` implementation anyway). I included the asynchronous doctests for stuff that is actually asynchronous (like `spawn`). If you want me to retain the async stuff everywhere (like `tokio_fs::File` does it), let me know.

Reply:
Maybe for the simple synchronous functions that just setup the environment and arguments we can simply link to the corresponding documentation from `std`?

Reply:
Yeah linking to the std docs is good, but this is fine too!